content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md

```diff
@@ -0,0 +1,149 @@
+---
+author: hiddeco
+date: 2023-12-12 16:00:00+00:00
+title: Announcing Flux 2.2 GA
+description: "Flux v2.2.0: Improving your Helm experience!"
```

Comment:
Suggestions are welcome, as long as it's within bounds of not being too enthusiastic.